wallet/lib/pages/home/home.dart

```diff
@@ -285,6 +320,19 @@ class HomeScreenState extends State<HomeScreen> with SingleTickerProviderStateMi
                     width: 20.w,
                   )),
             ),
+            if (remoteConfigService.getMaintenanceMode())
+              Positioned(
```

Comment:
Is it possible to create a widget called MaintenanceBanner and use it everywhere?